### PR TITLE
Add cache to enumerations when getting by mnemonic

### PR DESCRIPTION
**Pull request by arnaud-morvan**

### CODE REVIEW

**Comment by arnaud-morvan** [comment]
@pgiraud : could you review ?

File: thinkhazard_common/models.py [hunk 1]
```diff
@@ -33,10 +36,14 @@ class AdminLevelType(Base):
...
     def get(cls, mnemonic):
+        if mnemonic in adminleveltype_cache:
+            return adminleveltype_cache[mnemonic]
         with DBSession.no_autoflush:
-            return DBSession.query(cls) \
+            adminleveltype = DBSession.query(cls) \
                 .filter(cls.mnemonic == mnemonic) \
                 .one_or_none()
+            adminleveltype_cache[adminleveltype.mnemonic] = adminleveltype
+            return adminleveltype
```
Comment (pgiraud) [inline thinkhazard_common/models.py:54]: What about only one `return`?

``` python
if mnemonic not in adminleveltype_cache:
    with DBSession.no_autoflush:
        adminleveltype = DBSession.query(cls) \
            .filter(cls.mnemonic == mnemonic) \
            .one_or_none()
        adminleveltype_cache[mnemonic] = adminleveltype
return adminleveltype_cache[mnemonic]
```

Comment (arnaud-morvan) [inline thinkhazard_common/models.py:54]: I've encouter some problems with session expiration in tests, see next commit, so this should not apply anymore.

**Comment by pgiraud** [comment]
Please merge.

**Pull request merged**